getting-started/git-boot-camp.rst

```diff
@@ -283,6 +283,15 @@ Creating a Pull Request
 
 6. Press the ``Create pull request`` button.
 
+You should include the issue number in the title of the PR,
+in the format ``gh-NNNNN: <PR Title>``.
+
+You might also use `special keywords
+<https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword>`_
+to automatically close the linked issue once the PR is merged. However, you
+should avoid them if other PRs need to be merged before the issue can be
+closed (e.g. backports to other branches).
```

Comment:
For a new contributor, this might raise more questions than it answers. Can we link to a reference somewhere else in the devguide that tells contributors which PRs are generally backported, and which are not?

Reply:
To note, there's also other benefits to linking a PR beyond just auto-close—most notably, it adds an easy to find reference to the PR from the issue and the issue from the PR a number of handy places in the GitHub UI, which make navigation a fair bit quicker than manually/semi-automatically trying to figure out the corresponding PR (or issue).

Reply:
Nearly all bug fixes, doc fixes, and IDLE changes are backported, and nearly all feature changes are not.  But even feature issues may require multiple PRs, so the above advice is still incomplete.  And there are PRs without an issue.  By linking autoclose to linking of issue to PR, GH has made issue-PR linking worse than useless.

Reply:
Yeah, unfortunately the story on GitHub around relationships of multiple PRs for one issue has never been good, and unlike a number of other pain points that have improved substantially in recent years, this one doesn't seem to have. And unlike UI issues, there's not as much Refined GitHub and tools like it can do about it—it is a big reason that GitHub overlays like ZenHub gained such a following

Reply:
> unfortunately the story on GitHub around relationships of multiple PRs for one issue has never been good

I created this:
* https://github.com/python/bedevere/issues/501

Reply:
hey folks, I've captured this feedback! keep it coming :)

Reply:
Thanks @omerbensaadon ! We all really appreciate it!